doc/routinator.1

```diff
@@ -330,6 +333,26 @@ This format produces no output whatsoever.
 .TP
 .BR \-n ,\  \-\-noupdate
 The repository will not be updated before producing the list.
+.TP
+.BI \-a \ asn\fR,\  \-\-filter\-asn= asn
+Only output VRPs for the given ASN. The option can be given multiple times,
+in which case VRPs for all provided ASNs are provided. ASNs need to be given
+as the AS number prefixed with 
+.B AS 
+and not just the number
+e.g., the correct form is
+.BR "AS65555" .
+.TP 
+.BI \-p \ prefix\fR,\  \-\-filter\-prefix= prefix
+Only output VRPs with an address prefix that covers the given prefix, i.e.,
+whose prefix is equal or less specific than the given prefix. This will
+include VRPs irregardless of their ASN and max length. In other words, the
```

Comment:
I was taught "irregardless" is not a 'proper' word, and that "regardless" works just as well.

Reply:
It is not a word at all, and the etymologically extrapolated meaning is the opposite of how it is typically used (including in this instance).

Reply:
You are, of course, right.

Fixed by d628aeb.